app/main.py

```diff
@@ -1,3 +1,64 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> "Distance":
```

Comment:
Indicate that `other` could be a `Distance` class instance as well. Add corresponding changes to the other methods  
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```